package.json

```diff
@@ -19,6 +19,10 @@
     "export-app": "cd ./lib && npm run build && cd ../app && npm run build && npm run export",
     "dev-lib": "npm run dev --workspace=lib",
     "dev-site": "npm run build-lib && npm run dev --workspace=site",
-    "dev-app": "npm run build-lib && npm run dev --workspace=app"
+    "dev-app": "npm run build-lib && npm run dev --workspace=app",
+    "format-all": "prettier --write ./"
+  },
+  "dependencies": {
```

Comment:
```suggestion
  "devDependencies": {
```

Reply:
I removed prettier from the dependencies now. 
See the update in the description above and the mentioned commit [here](https://github.com/KlimaDAO/klimadao/pull/43#pullrequestreview-835702684)
Prettier is available already.